### PR TITLE
Rename struct fields 

### LINKED ISSUE #91: Replace CamelCase with Dashes
jsonapi now recommends to use dashes for naming of fields

> It is recommended that resource types, attribute names, and association names be "dasherized"; i.e. consist of only lower case alphanumeric characters and dashes (U+002D HYPHEN-MINUS, "-").

> It is also recommended that resource types be pluralized. Dasherized and pluralized resource types can be used as URL segments without translation, as discussed below.

### PR DESCRIPTION
**Pull request by sharpner**
Fixes #91

### CODE REVIEW

File: jsonapi/helpers.go [hunk 1]
```diff
@@ -79,3 +80,26 @@ func Pluralize(word string) string {
 func Singularize(word string) string {
 	return inflector.Singularize(word)
 }
+
+//GetTagValuesByName returns one api2go setting.
```
Comment (wwwdata) [inline jsonapi/helpers.go]: please one empty space after slashes

**Comment by wwwdata** [comment]
I would suggest to modify the `jsonapi.Dejsonify` and `jsonapi.Jsonify` methods too, so that they will encode CamelCase to dashes-case, like:

LargeGopher => large-gopher
GopherUUID => gopher-uuid (special case)

but we could also leave that out for now and if a user wants to use dashes, he has to use our tags

**Comment by wwwdata** [comment]
:+1:

**Comment by sharpner** [comment]
I strongly tend to make the magic renaming feature a nice to have for the future, because it is really complex to find a solution that fits every scenario.

**Comment by wwwdata** [comment]
yes, I agree. Let's update the README and then merge it

**Comment by sharpner** [comment]
done

**Pull request merged**